keyboards/foxlab/key65/universal/universal.h

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2020 Jumail Mundekkat / MxBlue
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_625u( \
```

Comment:
```suggestion
#define LAYOUT_65_ansi_blocker_split_bs( \
```

Reply:
Could you explain the reasoning behind the super specific name? "65" and "blocker" is redundant since that's all that the keyboard supports. Would also be worth noting that "blocker" is ambigious in this case since there's also the idea of the top right blocker.

Reply:
This layout is the same as the [`65_ansi_blocker_split_bs`](https://github.com/qmk/qmk_firmware/tree/4604c70c4c1a0cb2cebc010dba34b04e902bd982/layouts/default/65_ansi_blocker_split_bs) Community Layout. When possible, we prefer keyboards use the Community Layout names.

There's also [`65_ansi_blocker`](https://github.com/qmk/qmk_firmware/tree/4604c70c4c1a0cb2cebc010dba34b04e902bd982/layouts/default/65_ansi_blocker) and [`65_ansi_blocker_tsangan`](https://github.com/qmk/qmk_firmware/tree/4604c70c4c1a0cb2cebc010dba34b04e902bd982/layouts/default/65_ansi_blocker_tsangan). Within QMK, `LAYOUT_65_ansi_blocker` means this layout:

![image](https://user-images.githubusercontent.com/18669334/82273783-3cc2c280-9933-11ea-8335-38e06dbbbe8c.png)

... then QMK's Community Layouts also includes variants of that – one with Split Backspace, and the other with the Tsangan bottom row. The other layout here (the one that's currently `LAYOUT_7u`) combines those.

Reply:
Cool, will accept your changes. Also, does that mean I can add `LAYOUT = LAYOUT_65_ansi_blocker_split_bs LAYOUT_65_ansi_blocker_tsangan_split_bs LAYOUT_65_ansi_blocker_split_bs` to rules.mk too?

Reply:
> Also, does that mean I can add `LAYOUT = 65_ansi_blocker_split_bs 65_ansi_blocker_tsangan_split_bs 65_ansi_blocker_split_bs` to rules.mk too?

Set only `LAYOUTS = 65_ansi_blocker_split_bs`, unless you want to add layout macros for the other combinations supported by the PCB.

Having the layout macros with the preferred names, even if the community layouts don't exist, gives us flexibility to enable them by simply changing the value of `LAYOUTS` in `rules.mk`. If the layout macros need to be renamed or aliased, that's a much messier change.